### DISCUSSION TITLE
不知道发什么，发点爱莉希雅大人ᐠ( ᑒ )ᐟ 的照片

### DISCUSSION THREAD

**Discussion by WuxuRiver** [share121/inter-knot#5258] [其他]
![Image_1726415728335](https://github.com/user-attachments/assets/7bc3e501-7f45-4952-90ca-04c7d318315b)
![Image_1726415582994](https://github.com/user-attachments/assets/5fcfa6de-97e6-4e1b-8879-3217c4553254)
![img1451936502_1722323900813](https://github.com/user-attachments/assets/8832634e-5aa8-4fbe-bca8-112712c311b4)

**Comment by wshtdgo** [comment]
爱门！

**Comment by summer293** [comment]
爱莉希雅 ᐠ( ᑒ )ᐟ爱莉希雅 ᐠ( ᑒ )ᐟ

**Comment by 119009116** [comment]
爱门

**Reply by WuxuRiver** [reply]
> 爱门

千人律者，你该死啊！

**Reply by 119009116** [reply]
我说不是我干的，你信吗

**Reply by WuxuRiver** [reply]
> 我说不是我干的，你信吗

闲话就到此为止了！薪炎！拔剑！

**Reply by 119009116** [reply]
补药啊
